front/app/components/FilterBoxes/TopicsFilter.tsx

```diff
@@ -4,7 +4,9 @@ import {
   fontSizes,
   colors,
   defaultCardStyle,
+  Title,
 } from '@citizenlab/cl2-component-library';
+import CollapsibleContainer from 'component-library/components/CollapsibleContainer';
 import { isError, includes } from 'lodash-es';
```

Comment:
Same here - just used the new CollapsibleContainer here so we will have a visual match for all 3 filter boxes.